tests/Functional/Driver/PgSQL/BooleanBindingTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\PgSQL;
+
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+
+class BooleanBindingTest extends FunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (TestUtil::isDriverOneOf('pgsql')) {
+            $table = new Table('t');
+            $table->addColumn('v', 'boolean');
+            $this->dropAndCreateTable($table);
+
+            return;
+        }
+
+        self::markTestSkipped('This test requires the pgsql driver.');
```

Comment:
This test should pass with all drivers. Please move it to `tests/Functional` (see `BlobTest` for example).